docs/cities/manchester/roles-and-duties.md

```diff
@@ -0,0 +1,111 @@
+---
+id: roles-and-duties
+title: Roles & Duties
+---
+
+Under the main trinity of [Volunteer teams](volunteers/teams.md),namely Outreach, Education & Personal Development,
+generally, we need to act and fulfill different roles in order to run the course smoothly and effectively.
+These roles also help to keep each member being engaged and organised.
+
+A volunteer can wear different hats on the way depending on the schedule and agenda, or maybe special circumstances.
+All roles and tasks are optional, and no one is forced, or coerced into to doing things that they do not feel comfortable with.
+
+Any help is welcomed :) 
+
+## Roles
+Mostly, we need the following roles as below.
+Each role is charged of duties and demands various skill set.
+
+**Note:** This list is only a general advise, does not have to be strictly followed.
+
+### Meeting Manager
+ - Run the kick-off meeting at the start of the day with all volunteers
+ - Run retrospective at the end of the class
+ - Moderator the discussions in these meetings
+
+**Requirements**
+ - Good timekeeping
+ - Organised
+
+### Time keeper
+ - Keeps classes running on schedule
+ - Coordinates with module lead when tasks over run
+ - Gives heads up towards end of sessions
+
+**Requirements**
+ - Strict with time keeping
+ - Good time keeper
+ - Good coordination
+
+### Lead Instructor
```

Comment:
Is this the "Module Lead" role? If so, this only focuses on what they do in class and doesn't reflect what needs doing during the week to organise and coordinate what is to happen during a given module. I'm sure there are existing 'job descriptions' somewhere that could be pulled either into this page, or linked out as more expansive pages in their own right.